src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,88 @@
 package core.basesyntax;
 
+import static java.lang.Integer.parseInt;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
 
+        String createReportFull = readFromFile(fromFileName);
+        String createReport = createReport(createReportFull);
+        writeToFile(toFileName, createReport);
+    }
+
+    public String readFromFile(String fromFileName) {
+        File file = new File(fromFileName);
+        StringBuilder stringBuilder = new StringBuilder();
+
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(file));
```

Comment:
Buffered reader must be closed in "finally" block. But it's better to use "try with resources" for autoclosing
```suggestion
        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFile))) {
```